packages/integration-tests/src/test-e2e/backend_output.test.ts

```diff
@@ -0,0 +1,58 @@
+import { after, before, describe, it } from 'node:test';
+import { getTestProjectCreators } from '../test-project-setup/test_project_creator.js';
+import {
+  createTestDirectory,
+  deleteTestDirectory,
+} from '../setup_test_directory.js';
+import { fileURLToPath } from 'node:url';
+import { testConcurrencyLevel } from './test_concurrency.js';
+import { BackendIdentifier } from '@aws-amplify/plugin-types';
+import { TestProjectBase } from '../test-project-setup/test_project_base.js';
+import {
+  amplifySharedSecretNameKey,
+  createAmplifySharedSecretName,
+} from '../shared_secret.js';
+import { TestBranch, amplifyAppPool } from '../amplify_app_pool.js';
+
+const testProjectCreators = getTestProjectCreators();
+
+// Different root test dir to avoid race conditions with e2e deployment tests
+const rootTestDir = fileURLToPath(
+  new URL('../e2e-outputs-tests', import.meta.url)
+);
+
+void describe(
+  'backend output tests',
+  { concurrency: testConcurrencyLevel },
+  () => {
+    const testProjectCreator = testProjectCreators[0];
```

Comment:
rather than grabbing the first element of this array (which is arbitrarily ordered and someone might change it without realizing we're depending on the order here), maybe we should not use `getTestProjectCreators` in this suite and just construct the `DataStorageAuthWithTriggerTestProjectCreator` directly here?

Reply:
Good point, I'll do that in this PR